packages/components/src/organisms/PaymentMethods/PaymentMethods.tsx

```diff
@@ -53,13 +55,13 @@ const PaymentMethods = forwardRef<HTMLDivElement, PaymentMethodsProps>(
           data-fs-payment-methods-flags
           aria-label={title ? undefined : ariaLabel}
         >
-          {flagList.map((item, index) => (
+          {flagList.map(({ alt: text, icon: { icon } }, index) => (
             <li
               data-fs-payment-methods-flag
-              key={`fs-payment-method-${index}-${item.text}`}
+              key={`fs-payment-method-${index}-${text}`}
             >
-              {item.image}
-              {item.text && <SROnly text={item.text} />}
+              <Icon width={32} height={22.5} name={icon} />
```

Comment:
Where are these numbers from? 32 and 22.5

Reply:
This is what we had on the [FooterFlags](packages/core/src/components/common/Footer/FooterFlags.tsx) mock

Reply:
Will it still work if removing these sizes? 🤔

Reply:
Probably won't, but these styles we need to set anyway, don't we?